tests/testing_config/custom_components/test/valve.py

```diff
@@ -77,6 +77,14 @@ def set_valve_position(self, position: int) -> None:
         self._target_valve_position = position
         self.async_write_ha_state()
 
+    def stop_valve(self) -> None:
+        """Stop the valve."""
+        self._attr_is_closing = False
+        self._attr_is_opening = False
+        self._target_valve_position = None
+        self._attr_is_closed = self._target_valve_position == 0
```

Comment:
This will never be true if we set the `_target_valve_position` attribute to None above.

Reply:
True. I should have used the `_current_valve_position`, which AFAIK cannot be None. Not that it would matter a lot since this is a mock entity.